hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -109,19 +110,26 @@ public ErrorHandler errorHandler() {
         /**
          * {@link ConnectionFactory} with enabled publisher confirms and
          * heartbeat.
-         * 
+         *
          * @param config
          *            with standard {@link RabbitProperties}
          * @return {@link ConnectionFactory}
+         * @throws GeneralSecurityException
+         *             in case of problems with enabled SSL connections
          */
         @Bean
-        public ConnectionFactory rabbitConnectionFactory(final RabbitProperties config) {
+        public ConnectionFactory rabbitConnectionFactory(final RabbitProperties config)
+                throws GeneralSecurityException {
             final CachingConnectionFactory factory = new CachingConnectionFactory();
             factory.setRequestedHeartBeat(amqpProperties.getRequestedHeartBeat());
             factory.setExecutor(threadPoolExecutor);
             factory.getRabbitConnectionFactory().setHeartbeatExecutor(scheduledExecutorService);
             factory.setPublisherConfirms(true);
 
+            if (config.getSsl().isEnabled()) {
```

Comment:
At `SecurityProperties` for http it's called `requireSsl`. I would use the same naming